sdk/messaging/azservicebus/processor.go

```diff
@@ -33,27 +34,21 @@ type processorConfig struct {
 
 // Processor is a push-based receiver for Service Bus.
 type Processor struct {
-	// configuration data that is read-only after the Processor has been created
-	config processorConfig
+	*messageSettler
```

Comment:
Oh I just noticed this.  It means that this type is part of public surface area.  Assuming this was intentional, can you add a comment to `messageSettler` indicating that it's not really internal?

It would be ideal to avoid this if possible (mostly I worry about inadvertently introducing a breaking change to `messageSettler`).

Reply:
I think I'll just hide it for real, instead of doing what I'm doing here. Principle of least surprise and all that.

Reply:
Okay, done. messageSettler is no longer embedded.